core/src/display_object.rs

```diff
@@ -229,8 +230,6 @@ pub struct DisplayObjectBase<'gc> {
     /// Values other than the default `BlendMode::Normal` implicitly cause cache-as-bitmap behavior.
     blend_mode: Cell<ExtendedBlendMode>,
 
-    #[collect(require_static)]
-
```

Comment:
I don't know how this change is related or why it's necessary, haven't tested it.